docs/content/testing/writing-a-new-test.md

```diff
@@ -0,0 +1,314 @@
+[google-docs-link]: https://docs.google.com/document/d/14_FY-Chu_illhaCCym-pv6P8NHtyN9PLzVBGSuK0ou4/edit (Please place reviews as comments into this document here)
+
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Writing a new test
+
+This document covers Unity tests only (that is tests of the Unity project which forms part of the SpatialOS Unity GDK). It walks through creating tests for a made-up class to explain how you can test parts of the Unity GDK. You need to get to know NUnit and the Unity Test Runner before creating tests. See below for more information on these.
+
+## Prerequisites
+
+Please look through and understand the follow documentation and videos before writing a new test:
+* SpatialOS Unity GDK documentation - [Testing overview](./test-overview.md)
+* SpatialOS Unity GDK documentation - [How to run tests](./how-to-run-tests.md)
+* SptialOS Unity GDK documentation - [Testing guidelines](./test-guidelines.md)
+* NUnit video - [Introduction to NUnit](https://www.youtube.com/watch?v=1TPZetHaZ-A)<br/>
+**Skip** the part between 0:40 - 2:49 as it does not apply to Unity testing. <br/>
+**Skip** the part between 5:25 - 7:57 as it explains how to use the NUnit Runner and we’ll use Unity Test Runner instead.
+* NUnit video - [Unit Testing Using NUnit](https://www.codeproject.com/articles/178635/unit-testing-using-nunit)
+- Unity documentation - [Writing and executing tests in Unity Test Runner](https://docs.unity3d.com/Manual/PlaymodeTestFramework.html)
+* Infallible Code video -  [How To Test Unity ECS Code](https://http://infalliblecode.com/test-unity-ecs/) (Note the code for this is behind a paywall.)
```

Comment:
broken link, should be https://infalliblecode.com/test-unity-ecs/

Reply:
Https cert problems, reverting to http